requirements/optional.txt

```diff
@@ -0,0 +1 @@
+python-ldap==3.0.0 # optional for LDAP authentication, requires libldap (OpenLDAP) to build
```

Comment:
Maybe call this file "production", or is it always optional? Can you run in production without ldap without error?

Reply:
Yes, LDAP must be specifically enabled in the the config before the ldap library is even attempted imported anywhere. Even then, I think the code translates a failed import into a more user friendly error message about why stuff isn't working.